app/bundles/ReportBundle/Event/ReportBuilderEvent.php

```diff
@@ -318,6 +318,20 @@ public function getChannelColumns()
         return $channelColumns;
     }
 
+    /**
+     * @param array<string, mixed> $properties
+     *
+     * @return array<string, string|int>
+     */
+    public function getObjectColumns(string $object, array $properties = []): array
+    {
+        if ('contact' === $object || 'company' === $object) {
+            return [];
+        }
```

Comment:
It would be nice to refactor these core objects to se this event too instead of handling them some other way. This PR would then make more sense and would be testable.